mib-generator/cw_mib_generator.py

```diff
@@ -84,23 +99,24 @@ def generate_mib(common_dir, extras_dir, common_mib, extras_mib, output_mib):
     Args:
         common_dir (str): Directory containing common MIB fragment.
         extras_dir (str): Directory containing extra MIB fragment.
-        common_mib (str): File name of common MIB.
-        extras_mib (str): File name of extras MIB.
-        output_mib (str): File name for writing output MIB. NOTE: This will
-            over-write any existing file with the given name in extras_dir!
+        output_dir (str): Output directory for full MIB.
+        common_mib_name (str): File name of common MIB.
+        extras_mib_name (str): File name of extras MIB.
+        output_mib_name (str): File name for full MIB. NOTE: If this file
+            already exists in output_dir, it will be over-written!
```

Comment:
It's all very well noting that here, but you haven't noted it in the user documentation where it might save someone from making a mistake.